pyside6 is finally cached so it's possible to build this now. For some reasons i also get a `normcap.dist` package for free ?

When running normcap though it fails with this error:

```
15:20:57 - CRITICAL - normcap:161 - Uncaught exception! Quitting NormCap!
15:20:57 - CRITICAL - normcap:187 - Uncaught exception! Quitting NormCap! (debug output limited)
Traceback (most recent call last):
  File "/nix/store/8w24h7wh234d4ijwx4yki41ppz17wc6d-normcap-0.4.4/bin/.normcap-wrapped", line 9, in <module>
    sys.exit(main())
  File "/nix/store/8w24h7wh234d4ijwx4yki41ppz17wc6d-normcap-0.4.4/lib/python3.10/site-packages/normcap/app.py", line 57, in main
    tray = SystemTray(app, vars(args))
  File "/nix/store/8w24h7wh234d4ijwx4yki41ppz17wc6d-normcap-0.4.4/lib/python3.10/site-packages/normcap/gui/tray.py", line 60, in __init__
    logger.debug("System info:\n%s", system_info.to_dict())
  File "/nix/store/8w24h7wh234d4ijwx4yki41ppz17wc6d-normcap-0.4.4/lib/python3.10/site-packages/normcap/gui/system_info.py", line 165, in to_dict
    "tesseract_path": get_tesseract_path(),
  File "/nix/store/8w24h7wh234d4ijwx4yki41ppz17wc6d-normcap-0.4.4/lib/python3.10/site-packages/normcap/gui/system_info.py", line 81, in get_tesseract_path
    raise RuntimeError(
RuntimeError: No Tesseract binary found! Tesseract has to be installed and added to PATH environment variable.
15:20:57 - CRITICAL - normcap:192 - Please open an issue with the output above on https://github.com/dynobo/normcap/issues
```

Obviously it's missing the `tesseract` dependency, so you will have to add that.
To avoid more problems like that have a look at the AUR package at https://aur.archlinux.org/packages/normcap and its dependencies, this might help.

--- a/pkgs/applications/graphics/normcap/default.nix
+++ b/pkgs/applications/graphics/normcap/default.nix
@@ -1,0 +1,52 @@
+# From NUR https://github.com/nix-community/nur-combined/blob/6bddae47680482383b5769dd3aa7d82b88e6cbc8/repos/renesat/pkgs/normcap/default.nix
+
+{
+  lib,
+  stdenv,
+  python3,
+  fetchFromGitHub,
+  tesseract4,
+  leptonica,
+  wl-clipboard
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "normcap";
+  version = "0.4.4";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "dynobo";
+    repo = "normcap";
+    rev = "v${version}";
+    hash = "sha256-dShtmoqS9TC3PHuwq24OEOhYfBHGhDCma8Du8QCkFuI=";
+  };
+
+  buildInputs = [
+    wl-clipboard
+  ];
+
+  nativeBuildInputs = with python3.pkgs; [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    tesseract4
+    leptonica
+    pyside6
+
+    # Test
+    toml
+    pytest-qt
+  ];
+
+  postPatch = ''
+    substituteInPlace pyproject.toml --replace 'PySide6-Essentials = "6.5.1"' ""
+  '';
+
+  meta = with lib; {
+    description = "OCR powered screen-capture tool to capture information instead of images";
+    homepage = "https://dynobo.github.io/normcap/";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ cafkafk ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -34541,6 +34541,8 @@ with pkgs;
 
   nomacs = libsForQt5.callPackage ../applications/graphics/nomacs { };
 
+  normcap = callPackage ../applications/graphics/normcap { };
+
   notepad-next = libsForQt5.callPackage ../applications/editors/notepad-next { };
 
   notepadqq = libsForQt5.callPackage ../applications/editors/notepadqq { };

